crates/mockiato-codegen/src/parse/method_inputs.rs

```diff
@@ -114,22 +114,25 @@ impl MethodArg {
                         Ok(MethodArg {
                             ident: sanitize_method_ident(&pat_ident.ident),
                             ty: captured.ty,
-                            span
+                            span,
                         })
-                    },
-                    _ => {
-                        Err(Error::Diagnostic(Diagnostic::spanned(
-                        span,
-                        Level::Error,
-                        "Ignored arguments are not supported")))
-                    },
+                    }
+                    _ => Err(
+                        DiagnosticBuilder::error(span, "Ignored arguments are not supported")
+                            .build()
+                            .into(),
+                    ),
                 }
             }
-            _ => Err(Error::Diagnostic(Diagnostic::spanned(
-                span,
-                Level::Error,
-                "Only captured arguments are supported",
-            ).note("This error should never appear, because rustc already enforces these requirements"))),
+            _ => Err(
+                DiagnosticBuilder::error(span, "Only captured arguments are supported")
+                    .note(
+                        "This error should never appear, because rustc already enforces these \
```

Comment:
Add an issue creation link on unexpected errors.

Reply:
Addressed in 0211a29